.env.production

```diff
@@ -1,8 +1,8 @@
 # Refers to a production environment, which may be either train or prod
 
 # @PROD
-REACT_APP_API_URL=https://360Api1.gordon.edu/
-REACT_APP_ANALYTICS_ID=UA-3559968-3
+# REACT_APP_API_URL=https://360Api1.gordon.edu/
+# REACT_APP_ANALYTICS_ID=UA-3559968-3
 
 # @TRAIN
-# REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
\ No newline at end of file
+REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
```

Comment:
Why is this changing?

Reply:
Oh, that's because we updated develop after merging into Master last - I didn't check for that. I'll need to make a release branch instead and change this file. Good catch!